src/Arcus.Templates.ServiceBus.Queue/Arcus.Templates.ServiceBus.Queue.csproj

```diff
@@ -0,0 +1,62 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <!--#if (AuthoringMode)-->
```

Comment:
Copy paste from Web API template or not? 🤔

Reply:
No, this is needed so we don't put 'Arcus' related project information in the resulting project. 'AuthoringMode' means here that it is realted to the 'author' of the template and not the 'user'.

Reply:
👌